src/controls/deploy.ts

```diff
@@ -78,6 +78,34 @@ const deploy = async (
   });
 
   if (response.error) {
+    const error = response.error;
+
+    //used for handling zod type validation errors
+    if (error.isAxiosError && error.response && error.response.data) {
+      const errorData = error.response.data[0];
+      if (errorData && errorData.errors && errorData.errors.issues) {
+        const cacheSettingError = errorData.errors.issues.find(
+          (issue: { path: string[] }) => issue.path[0] === "cacheSetting"
+        );
+        const providerError = errorData.errors.issues.find(
+          (issue: { path: string[] }) => issue.path[0] === "provider"
+        );
+
+        if (cacheSettingError) {
+          console.log(
+            cWrap.br("Endpoint Setting Error For Path: " + config.path),
```

Comment:
This is kind of a weird error message, can we give something more actionable to the user? How do they resolve this?